<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CapacitorBankPlacement</class>
 <widget class="QDialog" name="CapacitorBankPlacement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>SystemID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bankplacementSID_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bankplacementUID_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Alias:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bankplacementAlias_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Catalog Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bankplacementCatType_comboBox">
       <item>
        <property name="text">
         <string>CapBank -20 kV -500 kVAr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CapBank -20 kV -250 kVAr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CapBank -20 kV -1000 kVAr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CapBank -20 kV -100 kVAr</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CapBank -20 kV -2500 kVAr</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bank Type: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bankplacementBankType_comboBox">
       <item>
        <property name="text">
         <string>Shunt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Serial</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>No Load Control: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bankplacementNLC_comboBox">
       <item>
        <property name="text">
         <string>Yes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Bank Control:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bankplacementBC_comboBox">
       <item>
        <property name="text">
         <string>No control</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Volt/VAr control</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Voltage regulation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Time switched control</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Phase Code:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bankplacementPC_comboBox">
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ABC</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Normal Section [1]:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="bankplacementNS_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Load Demand Model:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="bankplacementLDM_comboBox">
       <item>
        <property name="text">
         <string>Commercial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Culture</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EDC</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Education</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EMB</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Goverment</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HLS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hospital</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>IND</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Others</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>R/C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Recidential</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RES</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RLG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Utility</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VIP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AGR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Undefined</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Generation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Capacitor bank</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="bankplacementDescription_plainTextEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="capacitorbankplacement_buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
